test/e2e/specs/editor/editor__post-basic-flow.ts

```diff
@@ -160,16 +162,33 @@ describe( DataHelper.createSuiteTitle( 'Editor: Basic Post Flow' ), function ()
 		skipItIf( envVariables.VIEWPORT_NAME === 'mobile' )( 'Save draft', async function () {
 			await editorPage.saveDraft();
 		} );
+
+		it( 'Publish post', async function () {
+			publishedURL = await editorPage.publish();
+		} );
 	} );
 
-	describe( 'Publish', function () {
-		it( 'Publish and visit post', async function () {
-			const publishedURL: URL = await editorPage.publish( { visit: true } );
-			expect( publishedURL.href ).toStrictEqual( page.url() );
+	describe( 'View post', function () {
+		let newPage: Page;
+
+		beforeAll( async function () {
+			newPage = await browser.newPage();
+		} );
+
+		it( 'View published post', async function () {
+			// Check for `blog` and `post` query params, used for stats tracking.
+			const trackingPixelLoaded = newPage.waitForResponse(
+				/pixel\.wp\.com\/g\.gif.*blog=[\d]+.*&post=[\d]+/
```

Comment:
This regex is pretty specific about the 1) content and 2) structure of the request.

Reply:
Nice, I think it looks great!

**Question (very non-blocking):** I believe we should be able to know both the blog _and_ the post ID at this point: any reason to not just look for them exactly in the regex? (As opposed to any other characters.) I could be wrong, but it seems like a nice extra level of validation that _shouldn't_ add any more flakiness!

Reply:
We don't actually know the post ID! `EditorPage.publish` doesn't return the intercepted response, but rather just the URL of the published post. While we can do some regex hackery to extract the post ID, I don't think that's a reliable solution.

We **do** know the blogID though, so that's something we can check against.

Reply:
Awesome, seems like a great middle-ground to me! 🚀